digitalocean/database/resource_database_cluster.go

```diff
@@ -628,6 +628,15 @@ func setDatabaseConnectionInfo(database *godo.Database, d *schema.ResourceData)
 		}
 	}
 
+	if database.UIConnection != nil {
+		d.Set("host", database.UIConnection.Host)
+		d.Set("port", database.UIConnection.Port)
+		d.Set("uri", database.UIConnection.URI)
+		d.Set("database", database.UIConnection.Database)
+		d.Set("user", database.UIConnection.User)
+		d.Set("password", database.UIConnection.Password)
+	}
```

Comment:
This overwrites the normal connection info when `database.UIConnection` exists. Is that the intention? If not, we should extend the schema to include new attributes like `ui_host`, `ui_port`, etc...

Reply:
Previously, I thought we might not require those details and only Dashboard connection details are required in case of opensearch.